tests/benchmarks/test_subscriptions.py

```diff
@@ -24,3 +26,22 @@ async def _run():
             assert value.data["something"] == "Hello World!"
 
     benchmark(lambda: asyncio.run(_run()))
+
+
+@pytest.mark.benchmark
+def test_subscription_long_run(benchmark: BenchmarkFixture) -> None:
+    s = """
+    subscription {
+        longRunning
+    }
```

Comment:
sorry to be pedantic, but could we make the subscription take a param? so we can test with 100 and 10_000 in one single test? :D

Reply:
np. though I don't think I understand wym in a single tests?

Reply:
similar to this: https://github.com/strawberry-graphql/strawberry/blob/main/tests/benchmarks/test_complex_schema.py#L89-L101